html/base/inputs/Radio.stories.js

```diff
@@ -22,9 +22,7 @@ export const defaultStory = () => {
     <div class="sprk-b-InputContainer">
       <fieldset class="sprk-b-Fieldset">
         <legend class="sprk-b-Legend">
```

Comment:
Good catch! Need to add the label class to the legend to now that the label is removed. So it can still get styling. `sprk-b-Label`